keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -579,20 +606,21 @@ to CNP rules.
 
 ![Alt text](user_story_diagram.png?raw=true "User Story Diagram")
```

Comment:
Do we need this image anymore?

Reply:
Isn't the user stories in the image still valid though? I'm not sure what's changed during my leave so please correct me if I'm wrong

Reply:
I mean we have the explicit diagrams for each user story now, and this diagram doesn't really seem to correspond to the actual sample specs at all (in terms of naming conventions etc)

Reply:
Got it, removing the diagram